cargo-shuttle/src/provisioner_server.rs

```diff
@@ -333,7 +333,7 @@ fn db_type_to_config(db_type: Type) -> EngineConfig {
     match db_type {
         Type::Shared(SharedEngine::Postgres) => EngineConfig {
             r#type: "shared_postgres".to_string(),
-            image: "docker.io/library/postgres:11".to_string(),
```

Comment:
This tries to match the same version that is being provisioned for RDS instances to make local and prod identical. I might have missed when the RDS instances also got bumped?

Reply:
I checked in the console now and a bit more than half of our RDS instances are at 14 (the newer ones), and the older half is at 13. I don't see any below 13, and the most recent instance is actually 15. :smile: We don't explicitly set the version in provisioner when we provision RDS dbs, so perhaps it just uses `latest` :thinking: